seqr/utils/logging_utils.py

```diff
@@ -85,4 +86,16 @@ def log_model_bulk_update(logger, models, user, update_type, update_fields=None)
         db_update['updateFields'] = list(update_fields)
     logger.info(
         '{} {} {}s'.format(update_type, len(entity_ids), db_entity), user, db_update=db_update)
-    return entity_ids
\ No newline at end of file
+    return entity_ids
+
+
+def log_model_no_guid_bulk_update(logger, models, user, update_type):
+    if models:
```

Comment:
It could be done. But to keep the caller function simple and with no changes to the return values of the caller function, it looks neater to check the models here.

Reply:
having the entire body of a function nested inside an if block is not neat. If you really need to do this check in this function, you should explicitly return when there are no models and then have the actual function code executed at the top level